index.html

```diff
@@ -125,6 +125,30 @@
           "!URL#concept-url-equals">equals</a>. [[!URL]]
         </p>
       </section>
+      <section>
+        <h2>
+          Fetching (dereferencing)
+        </h2>
+        <p>
+          As <a>URL-based payment method identifiers</a> serve primarily as
+          identifiers (and not a linked resource), it is OPTIONAL for user
+          agents to <a data-cite="!WHATWG-FETCH#concept-fetch">fetch</a> a
+          <a>URL-based payment method identifier</a>.
+        </p>
+        <p class="note">
+          The working group is in the early stages of standardizing a <a href=
+          "https://w3c.github.io/payment-method-manifest/">Payment
+          Manifest</a>, which will allow developers to describe their payment
+          methods in a machine-processable format. We encourage developers to
+          experiment with Payment Manifest by placing one at the <a>URL-based
```

Comment:
This is incorrect. The manifest should be placed at a location that is accessed using the algorithm described at https://w3c.github.io/payment-method-manifest/#accessing